client/packages/common/src/ui/components/footer/ActionsFooter.tsx

```diff
@@ -61,30 +45,18 @@ export const ActionsFooter: FC<ActionsFooterProps> = ({
       >
         {selectedRowCount} {t('label.selected')}
       </Typography>
-      {actions.map(
-        ({
-          label,
-          icon,
-          onClick,
-          disabled,
-          shouldShrink,
-          disabledToastMessage,
-        }) => (
-          <div
-            key={label}
-            onClick={handleDisabledClick(disabled, disabledToastMessage)}
-          >
-            <FlatButton
-              startIcon={icon}
-              label={label}
-              disabled={disabled}
-              onClick={onClick}
-              // Flatbutton doesn't shrink by default but we want it to in actions footer
-              shouldShrink={shouldShrink ?? true}
-            />
-          </div>
-        )
-      )}
+      {actions.map(({ label, icon, onClick, disabled, shouldShrink }) => (
+        <div key={label}>
+          <FlatButton
+            startIcon={icon}
```

Comment:
I can really see the purpose of this `div`, you should be able to just pass `key` to `FlatButton` to appease react 
```suggestion
          <FlatButton
            key={label}
            startIcon={icon}
```

Reply:
@roxy-dao am I correct in stating that this was needed to right align the clear selection button?

Reply:
No.... this line of code `justifyContent: 'space-between'` the key was added to fix console error about how keys should be unique